pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -78,7 +78,8 @@ func (p Provisioner) Provision(options *apibkt.BucketOptions) (*bktv1alpha1.Obje
 		return nil, errors.Wrapf(err, "Provision: can't create ceph user")
 	}
 
-	s3svc, err := NewS3Agent(p.accessKeyID, p.secretAccessKey, p.storeDomainName)
+	s3svc, err := NewS3Agent(p.accessKeyID, p.secretAccessKey, fmt.Sprintf("%s:%d", p.storeDomainName, p.storePort))
```

Comment:
This pattern `fmt.Sprintf("%s:%d", p.storeDomainName, p.storePort)` is used several times. I would suggest breaking this into a method on  the `Provisioner` type to reduce duplication and increase clarity. Something like below, though I'm not sure if this name is the best one.

```go
func (p Provisioner) storeAddress() string {
    return fmt.Sprintf("%s:%d", p.storeDomainName, p.storePort)
}
```

Reply:
Sure, this change should be what you're looking for.